frontend/src/components/Scratch/ScratchToolbar.tsx

```diff
@@ -173,7 +173,7 @@ function Actions({ isCompiling, compile, scratch, setScratch, setDecompilationTa
                     Fork
                 </button>
             </li>
-            {scratch.owner && userIsYou(scratch.owner) && <li>
+            {((scratch.owner && userIsYou(scratch.owner)) || api.useThisUserIsAdmin()) && <li>
```

Comment:
Don't call hooks conditionally! We should do
```
const isAdmin = api.useThisUserIsAdmin();
// ...
{((scratch.owner && userIsYou(scratch.owner)) || isAdmin)
```

Reply:
why?

Reply:
okay, I looked it up and there's information about it here: https://legacy.reactjs.org/docs/hooks-rules.html